.github/workflows/pr.yaml

```diff
@@ -41,6 +43,39 @@ jobs:
     uses: rapidsai/shared-workflows/.github/workflows/checks.yaml@branch-24.10
     with:
       enable_check_generated_files: false
+  fetch-pandas-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      pandas-versions: ${{ steps.get-versions.outputs.versions }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+
+      - name: Install requests and packaging
+        run: pip install requests packaging
+
+      - name: Fetch pandas versions
+        id: get-versions
+        run: |
+          versions=$(python ci/cudf_pandas_scripts/fetch_pandas_versions.py ">=2.0,<2.2.3dev0")
```

Comment:
Is it possible to use `dependencies.yaml`/RDFG (https://github.com/rapidsai/cudf/blob/branch-24.10/dependencies.yaml#L573) instead of hardcoding the version here?

Reply:
Done 👍